testutil/compose/fuzz/beacon_fuzz_test.go

```diff
@@ -20,14 +20,14 @@ import (
 //go:generate go test . -run=TestBeaconFuzz -integration -v
 
 var (
-	integration = flag.Bool("integration", false, "Enable docker based integration test")
+	beaconFuzz  = flag.Bool("beacon-fuzz", false, "Enable docker based integration test")
```

Comment:
I'd rather just use `fuzz` to allow other types of fuzzing, not just beacon node fuzzing.

Reply:
sure